src/request.ts

```diff
@@ -938,17 +940,27 @@ class DatastoreRequest {
    *   const apiResponse = data[0];
    * });
    */
-  save(entities, gaxOptions?, callback?) {
+  save(entities: Entities, gaxOptions?: CallOptions): void|Promise<google.datastore.v1.ICommitResponse>;
```

Comment:
This should probably just be returning a promise, not void

Reply:
Okay!